plugin/webpack.config.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+
+const webpack = require( 'webpack' );
+
+const config = module.exports = {
+	entry: {
+		app: './src/index.js'
```

Comment:
I was thinking we should call this folder `editor`, and it would more closely represent what we may be bringing into core.